LICENSE

```diff
@@ -1,3 +1,5 @@
+The MIT License (MIT)
+
 Copyright (c) 2014 OpenTable, Inc.
```

Comment:
I'm not sure if the other copyright holders should be added here?  Just because they're not, doesn't mean there aren't any though.